libs/application/template-api-modules/src/lib/modules/templates/health-insurance/health-insurance.utils.ts

```diff
@@ -23,6 +25,27 @@ const extractAnswerFromJson = (object: unknown, key: string) => {
   return JSON.parse(value)
 }
 
+export const parseNationalRegistryDate = (dateString: string | null) => {
+  if (typeof dateString !== 'string') {
+    return undefined
+  }
+
+  // Icelandic format
+  let parsedDate = parse(dateString, 'd.M.yyyy HH:mm:ss', new Date())
+  if (!isNaN(parsedDate.getTime())) {
```

Comment:
Will this ever not be true? Since we pass in a fallback date initialised to now.

Thinking if we should rather extract the default date to a variable `const defaultDate = new Date(0)`

And in both of these checks look if `parsedDate.getTime() !== 0`?